README.md

```diff
@@ -20,26 +36,34 @@ as well as a match on one additional field above, or on
+
+## Generating merged articles
+
+Given a set of articles that have been matched together, we [generate](utils/create_merge_ids.py) a single "merged id"
+that is linked to all the "original" (vendor) ids of those articles. Some points from our implementation:
 
-To do this, we run the `linkage_dag.py` on airflow. The article linkage runs weekly, triggered by the `scholarly_lit_trigger` dag.
+* If articles that have been seen in a previous run and were previously assigned to different merged ids are now matched
+together, we assign them to a new merged id.
+* If a set of articles previously assigned to a given merged id _loses_ articles (either because it is now assigned to
+a different merged id, or because it has been deleted from one of the input corpora), we give this set of articles a
+new merged id.
+* If a set of articles previously assigned to a given merged id _gains_ articles without losing any old articles, we
+keep the old merged id for these articles.
 
-For an English description of what the dag does, see [the documentation](methods_documentation/overview.md).
+This implementation is meant to ensure that downstream pipelines (e.g. model inference, canonical metadata assignment)
+always reflect outputs on current metadata for a given merged article regardless of downstream pipeline implementation.
 
-### How to use the linkage tables (CSET only)
+## Automation and output tables
 
-We have three tables that are most likely to help you use article linkage.
+We automate article linkage using Apache Airflow. `linkage_dag.py` contains our current implementation.
 
-- `gcp_cset_links_v2.article_links` - For each original ID (e.g., from WoS), gives the corresponding CSET ID.
-This is a many-to-one mapping. Please update your scripts to use `gcp_cset_links_v2.article_links_with_dataset`,
-which has an additional column that contains the dataset of the `orig_id`.
+* This dag is triggered from the [Semantic Scholar ETL dag](https://github.com/georgetown-cset/semantic-scholar-etl-pipeline/blob/main/s2_dag.py) which runs once a month.
+* This dag triggers the [Org Fixes dag](https://github.com/georgetown-cset/org-fixes/blob/main/org_fixes_dag.py).
 
-- `gcp_cset_links_v2.all_metadata_with_cld2_lid` - provides CLD2 LID for the titles and abstracts of each
-current version of each article's metadata. You can also use this table to get the metadata used in the
-match for each version of the raw articles. Note that the `id` column is _not_ unique as some corpora like WOS
-have multiple versions of the metadata for different languages.
+The DAG generates two tables of analytic significance:
 
-- `gcp_cset_links_v2.article_merged_metadata` - This maps the CSET `merged_id` to a set of merged metadata.
-The merging method takes the maximum value of each metadata field across each matched article, which may not
-be suitable for your purposes.
+* `staging_literature.all_metadata_with_cld2_lid` - captures metadata for all unmerged articles in a
```

Comment:
Potentially highlight what the metadata here looks like - for example, how to select titles/abstracts that are linked but might be different (my recollection is we take the longest available one), keeping the original language title/abstract if available, etc. External readers won't have the ability to go look at what's in the table.

Reply:
Metadata selection happens downstream in https://github.com/georgetown-cset/cset_article_schema so I think this documentation belongs there, but I'll add a link to that repo which should help clarify that this work happens elsewhere.